pkg/apis/config/v1alpha1/mesh_config.go

```diff
@@ -25,6 +25,7 @@ type MeshConfigSpec struct {
 type SidecarSpec struct {
 	EnablePrivilegedInitContainer bool   `json:"enablePrivilegedInitContainer,omitempty" yaml:"enablePrivilegedInitContainer,omitempty"`
 	LogLevel                      string `json:"logLevel,omitempty" yaml:"logLevel,omitempty"`
+	EnvoyImage                    string `json:"envoyImage,omitempty" yaml:"envoyImage,omitempty"`
```

Comment:
You will need to run `make codegen` to update the `zz_generated.deepcpoy.go` file as well

Reply:
EnvoyImage does not need to be added to `zz_generated.deepcopy.go` so `make codegen` will not update the file